tests/rptest/services/verifiable_consumer.py

```diff
@@ -215,6 +216,8 @@ def __init__(self,
         self.stop_timeout_sec = stop_timeout_sec
         self.on_record_consumed = on_record_consumed
         self.verify_offsets = verify_offsets
+        self.last_committed = datetime.now()
+        self.last_consumed = datetime.now()
```

Comment:
Would it make more sense for the initial values to be `None`? It would be a bit weird for the log to contain actual timestamps when nothing was committed/consumed.

Reply:
good point, will change this